scripts/stack/README.md

```diff
@@ -0,0 +1,135 @@
+[Connectors CLI](../../connectors/connectors_cli.py) to create a new index and connector.
+
+The resulting configuration will be saved to the [config.yml](./connectors-config/config.yml)
+file that is used to start the Connectors container. Note that using the configurator,
+it will modify the [connectors config.yml](./connectors-config/config.yml) file. You can restore it
+by copying over the [example config.yml.example](./connectors-config/config.yml.example) file
+to `config.yml` in the directory.
+
+If you configured a new connector, you will need to go into Kibana to start or schedule a sync.
+Running the scripts, you can log in at http://localhost:5601/ with the username `elastic` and your password (`changeme` is the default, unless you specified a different `ELASTIC_PASSWORD` earlier).
+Then, to complete this in Kibana:
+* From the main menu, choose "Content" under the "Search" heading
+* Click on your index name
+* Click on the "Configuration" tab
+  * Note, the API key will be already pre-configured, and you do not need to generate a new one
+  * Optionally, change the name or add a description
+  * Ensure your connector settings are the way you want, if not edit them to their correct values
+  * Finally, set your schedule and sync
+
+Note - some of the connectors require a valid Elasticsearch license above `basic`.
+To see which connectors require a higher license tier, see the [Connectors reference](https://www.elastic.co/guide/en/enterprise-search/current/connectors-references.html).
+
+## Manual Configuration
+
+Instead of using the CLI based configurator while running the `run-stack.sh`
+script, you can choose to manually provide your connectors configurations and
+use those instead. To do this and still use the `run-stack.sh` script:
+
+1. Run the stack without the connectors service:
+    1. Run `./scripts/stack/run-stack.sh --no-connectors` to run the full stack without the Connectors service
+    2. From the main menu, go to "Content", "Indices", and create a new index for your connector
+    3. Choose the "Connector" index type
+    4. Select the connector type
+    5. Set the index name
+    6. Generate a new API Key, and copy it somewhere safe
+    7. Set Connector name and description (optional)
+    8. Copy the resulting connector_id, service_type, and api_key into a new entry in the `./scripts/stack/connectors-config/config.yml` fille
+    9. Repeat as necessary for all the connectors you wish too create
+3. run `./scripts/stack/run-stack.sh --no-configuration --connectors-only` again to run the Connectors service without prompting for configuration. Once running:
+    1. From the main menu, choose "Content" under the "Search" heading
+    2. Click on your index name
+    3. Click on the "Configuration" tab
+    4. Complete the configuration and optionally start your sync
+    5. Repeat these steps for all the connectors you created
+
+## Script command reference
```

Comment:
@acrewdson had mentioned about adding a script to conveniently accessing the logs earlier - since you're the second person who mentioned along these lines, I'll add one... but basically, you can do:
```
docker-compose -f ./scripts/stack/docker/docker-compose.yml logs -f elastic-connectors
```
As the container name ("elastic-connectors") should be able to be used as an alias for the id

Reply:
I ended up googling and finding that `docker logs docker-elastic-connectors-1 --follow` worked, too 🤷